frontend/src/services/preferences.js

```diff
@@ -34,6 +34,22 @@ class Preferences {
   set maxIdleTime(value) {
     this.__set__("max-idle-time", value);
   }
+
+  toggleVisibility(jobName) {
+    let hiddenJobs = JSON.parse(this.__get__("hidden-jobs") || "[]");
```

Comment:
we can use **const** instead of let.

Also can we pull _JSON.parse(this.__get__("hidden-jobs") || "[]");_ into a another method

Reply:
Changed accordingly